gui/cloudconnectionwizard.h

```diff
@@ -127,6 +129,7 @@ class CloudConnectionWizard : public Dialog {
 	CloudConnectionWizard();
 	~CloudConnectionWizard() override;
 
+	int runModal(uint32 selectedStorageIndex);
```

Comment:
This uses the same name as the base class, but it does not override it, which is why you added that `using` statement above.

It would be preferable to give this method a different name instead, e.g. `runStorageModal()`, to avoid unnecessary name clashes